### ISSUE TITLE
Disabled extensions are automatically updated by VS Code

### ISSUE THREAD

**Issue by onlineservices283418** [Microsoft/vscode#49259]
Issue Type: <b>Bug</b>

Disabled extensions are automatically updated by VS Code. The problem is that when a broken extension is released (for instance, https://github.com/Microsoft/vscode-chrome-debug/issues/657), a user has two ways to go:

1) temporarily uninstall the extension in question
2) turn off auto updates for all extensions

Each of the options has its downsides. I suggest not updating disabled extensions as they are not used anyway. VS Code should start updating extensions only after they get enabled.

VS Code version: Code 1.23.0 (7c7da59c2333a1306c41e6e7b68d7f0caa7b3d45, 2018-05-03T16:44:55.614Z)
OS version: Windows_NT x64 10.0.16299

**Comment by sandy081** [comment]
Duplicate of #22461
